LondonBritishLibrary/orient/BLorient811.xml

```diff
@@ -52,48 +310,70 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                                 </support>
                                 <extent>
                                     <measure unit="leaf">180</measure>
-                                    <dimensions type="outer" unit="mm">
-                                        <height>346</height>
-                                        <width>298</width>
+                                    <measure unit="leaf" type="blank">2</measure>
+                                    <dimensions type="outer" unit="in">
+                                        <height>13.625</height>
+                                        <width>11.75</width>
                                     </dimensions>
                                 </extent>
-                                
-                                
+                                <foliation>Two unnumbered blank leaves at the end.</foliation>
+
                             </supportDesc>
-                            
+
                             <layoutDesc>
-                                <layout columns="3" writtenLines="31" cert="high"/>
+                                <layout columns="3" writtenLines="31"/>
                             </layoutDesc>
                         </objectDesc>
-                        
+
+
                         <handDesc>
-                            
+
                             <handNote script="Ethiopic" xml:id="h1">
-                                <desc>The manuscript is well written  apparently in <date>1755</date>
+                                <desc>The manuscript is well written apparently in <date>1755</date>
```

Comment:
Is this date given in the manuscript? Is it the intersection of the mentioned persons' lifespans?

Reply:
This is indeed an intersection of king's reigns. The date has been proposed by Wright.